src/index.js

```diff
@@ -0,0 +1,48 @@
+import Tasks from './modules/index.js';
+import './style.css';
+
+const inTsk = {};
```

Comment:
- I think it would be better if you use more descriptive variable names. It helps your future self, your fellow coders, and everyone who stumbles upon your code without knowing the context 👍